pom.xml

```
@@ -192,7 +192,7 @@
                     <plugin>
                     	<groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
-                        <version>${surefire-plugin.version}</version>
+                        <version>3.1.2</version>
```

Since the `maven-surefire-plugin` version is used in multiple locations, we can use the `surefire-plugin.version` property in one location:

```suggestion
                        <version>${surefire-plugin.version}</version>
```

---

Good point! I have commited a fix for these comments 🚀